app/src/main/java/at/aau/edu/wizards/gameModel/GameModelBoard.kt

```diff
@@ -0,0 +1,106 @@
+package at.aau.edu.wizards.gameModel
+
+class GameModelBoard : GameModelBoardInterface {
+
+    private var trump: String? = null
+    private var winningCard: GameModelCard? = null
+    private val game = ArrayList<GameModelCard>()
+    private val winningCards = ArrayList<GameModelCard>()
+    private val trumpStack = ArrayList<String>()
+
+    override fun addTrumpToStack(trump: String): GameModelResult<Unit> {
+        if ((trump.length != 2 || trump[0].code > 14 || (trump[1].code !in 1..4 && (trump[0].code != 0 || trump[1].code != 0)))) {
```

Comment:
As personally discussed, this is a bit hard to follow in code. Let's create a future backlog task to work with more meaningful entities in code. For sending/receiving data it's fine to use very limited structure. But we don't need to rely on that structure at any other place.

Reply:
Structure changed to a sealed class representation in new iteration